test cases/common/48 test args/copyfile.py

```diff
@@ -3,4 +3,15 @@
 import sys
 import shutil
 
+# If either of these use `\` as the path separator, it will cause problems with
+# MSYS, MSYS2, and Cygwin tools that expect the path separator to always be
+# `/`. All Native-Windows tools also accept `/` as the path separator, so
+# it's fine to always use that for arguments.
+# See: https://github.com/mesonbuild/meson/issues/1564
+if '\\' in sys.argv[1]:
+    raise AssertionError('Found \\ in {!r}'.format(sys.argv[1]))
+
+if '\\' in sys.argv[2]:
+    raise AssertionError('Found \\ in {!r}'.format(sys.argv[2]))
```

Comment:
Nit: `Found \\ in destination argument {!r}`

Might save some time debugging later.

Reply:
Makes sense, will change, thanks. Ditto for the other.